ember-toucan-core/src/components/form/checkbox-field.ts

```diff
@@ -58,4 +80,16 @@ export default class ToucanFormCheckboxFieldComponent extends Component<ToucanFo
     assert('A "@label" argument is required', args.label);
     super(owner, args);
   }
+
+  get isChecked() {
+    if (!this.args?.option) {
+      return this.args.value;
+    }
+
+    if (!this.args.selectedValues || this.args.selectedValues?.length === 0) {
+      return false;
+    }
+
+    return this.args.selectedValues?.includes(this.args.option);
```

Comment:
This is where things get a little trickier in a checkbox group.  Essentially we need to check if the `@option` is in the `@selectedValues` array to determine if this particular checkbox is checked or not.